File: examples/java/modules/MathUtil.java
// 同一个包内的另一个类
public class MathUtil {
    public static final int MAX_SCORE = 100;        // public：包外可见

    public static double average(int[] scores) {
        if (scores.length == 0) return 0;
        int sum = 0;
        for (int s : scores) sum += s;
        return (double) sum / scores.length;
    }

    static String packagePrivate() {                // 无修饰符：仅包内可见
        return "package-private：只有同包能调用";
    }

    private static String hidden() { return "private：只有本类能调用"; }
}

File: examples/python/modules/mathutil.py
"""被导入的模块。模块本身就是一个对象。"""
print("   [mathutil.py 被执行了]")

__all__ = ["average", "MAX_SCORE"]      # 声明 from mathutil import * 导出什么

_internal = "单下划线：约定为内部使用"   # 约定私有
MAX_SCORE = 100

def average(scores):
    return sum(scores) / len(scores) if scores else 0

if __name__ == "__main__":
    print("直接运行时才执行；被 import 时不执行")
